zerver/webhooks/jotform/fixtures/response.multipart

```diff
@@ -65,7 +65,7 @@ Content-Disposition: form-data; name="customBody"
 --------------------------J1JNT4vbVO4FCC1qLG4HSv
 Content-Disposition: form-data; name="rawRequest"
 
-{"slug":"submit\/243231271343446","jsExecutionTracker":"build-date-1732271172685=>init-started:1732615894703=>validator-called:1732615894706=>validator-mounted-false:1732615894706=>init-complete:1732615894707=>onsubmit-fired:1732615897940=>observerSubmitHandler_received-submit-event:1732615897941=>submit-validation-passed:1732615897942=>observerSubmitHandler_validation-passed-submitting-form:1732615897944","submitSource":"form","buildDate":"1732271172685","uploadServerUrl":"https:\/\/upload.jotform.com\/upload","eventObserver":"1","q3_studentsName":{"first":"Niloth","last":"P"},"q33_typeOf":"Online Tutoring","q34_subjectFor":"Math","q35_grade":"12","q28_appointment":{"implementation":"new","date":"","duration":"60","timezone":"Asia\/Calcutta (GMT+06:30)"},"timeToSubmit":"3","preview":"true","validatedNewRequiredFieldIDs":"{\"new\":1}","path":"\/submit\/243231271343446"}
```

Comment:
If we're updating this fixture (the only fixture for Jotform), we'd also need to update the example screenshot in the [integration doc](https://zulip.com/integrations/doc/jotform).

When I ran the generate command for this fixture, it doesn't load the image properly, probably because of lazy loading of the image, not sure.
If you could get the image to load, and hide the tooltip, (by editing [the generator script](https://github.com/zulip/zulip/blob/main/tools/screenshots/generate-integration-docs-screenshot) as an initial commit) we could use this fixture for the screenshot.
Otherwise, we could create a separate fixture file for the files use case, and a separate test.
So that we don't edit the fixture that's used for the screenshot.

Feel free to duplicate this fixture though, because I plan to be updating the fixture used for the screenshot, to a business use case instead, so variety wouldn't be an issue.

Reply:
Actually, while manual testing too, if you open the localhost in a browser where you're not logged in with jotform, the images aren't being rendered. Maybe because those form uploads are confidential (yeah obv :smile: )
So, I too couldn't render the images using the generator script. Though, I have manually took screenshot of the webhook message carefully and uploaded it :)